site/documentation/content/dev-guide/amqp_adapter_client.md

```diff
@@ -0,0 +1,113 @@
++++
+title = "AMQP Adapter Client for Java"
+weight = 390
++++
+
+Eclipse Hono&trade; comes with a Java client for the AMQP adapter. It is intended to be used to implement (prototypical)
+devices, (protocol) gateways or to be used for (end-to-end) testing. It is based on Eclipse Vert.x.
+
+The starting point is the class `AmqpAdapterClientFactory`.
+
+It provides methods to get sender or receivers for the communication with the AMQP adapter. A sender is available for 
+each of the following actions:
+
+ * send telemetry message
+ * send event message
+ * send a response to a previously  received command
+ 
+
+{{% note title="Do not re-use sender instances" %}}
+The senders manage the underlying AMQP sender links, which are restored after a temporary interruption of the connection. 
+('HonoConnection' handles automatic reconnection). To achieve this, a caching mechanism is used, so that 
+ defective links are replaced by new ones. It is important that the sender is always retrieved from the factory before sending.
+ Do not keep links to sender objects and reuse them for subsequent send operations!
+{{% /note %}}
+
+
+## Create a Client Instance 
+
+```
```

Comment:
please do not put code into the docs. We will never be able to keep this in sync with changes that we make to the code itself. Please refer to example classes or test cases in the code base instead.